src/fu-tool.c

```diff
@@ -3532,6 +3561,15 @@ main(int argc, char *argv[])
 	     N_("Filter with a set of device flags using a ~ prefix to "
 		"exclude, e.g. 'internal,~needs-reboot'"),
 	     NULL},
+	    {"save-backends",
```

Comment:
don't forget to add bash completion for this

Reply:
Good catch, fixed.